pkg/recommender.py

```diff
@@ -441,21 +441,44 @@ def _schedule_evt(self):
     Send the morning message at 10 am
     '''
     next_morning = ''
+    schedule_evts = [(10,'999'),(23,'998')] #(hour, event_id)
+    #early to late
+    evt_count = 0
```

Comment:
add the following below, I notice the M2G also supports `minute`, so you probably will have strings like `23:00`
so you may need use delta `[(timedelta(hour=10, minute=0), '999')], (timedelta(hour=23, minute=0), '998')]

The following code will help the `evt_count` jump to the next event
```py
start_today = datetime.now().replace(hour=0, minute=0, second=0, microsecond=0)
for delta, _ in schedeule_evts:
  if start_today + delta < datetime.now():
    evt_count += 1
  else:
    break
```